# biodbKegg

[![Build Status](https://travis-ci.org/pkrog/biodbKegg.svg?branch=master)](https://travis-ci.org/pkrog/biodbKegg)

An R package for accessing [Kegg](https://www.kegg.jp/) online database, based on R package/framework [biodb](https://github.com/pkrog/biodb/).

## Introduction

This package implements *biodb* connectors for the following KEGG databases:
 * KEGG Compound.
 * KEGG Enzyme.
 * KEGG Genes.
 * KEGG Module.
 * KEGG Orthology.
 * KEGG Pathway.
 * KEGG Reaction.

## Examples

TODO

## Installation

TODO

## Documentation

TODO

## Citations

 * Kanehisa, M.; Toward pathway engineering: a new database of genetic and molecular pathways. Science & Technology Japan, No. 59, pp. 34-38 (1996), <http://www.kanehisa.jp/docs/archive/stj.pdf>.
 * Kanehisa, M.; A database for post-genome analysis. Trends Genet. 13, 375-376 (1997), <https://doi.org/10.1016/S0168-9525(97)01223-7>.
 * Kanehisa, M. and Goto, S.; KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 28, 27-30 (2000), <https://doi.org/10.1093/nar/28.1.27>.
 * Kanehisa, M. and Goto, S.; KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 28, 27-30 (2000), <https://doi.org/10.1093/nar/28.1.27>.
 * Kanehisa, M., Goto, S., Kawashima, S., and Nakaya, A.; The KEGG databases at GenomeNet. Nucleic Acids Res. 30, 42-46 (2002), <https://doi.org/10.1093/nar/30.1.42>.
 * Kanehisa, M., Goto, S., Kawashima, S., Okuno, Y., and Hattori, M.; The KEGG resources for deciphering the genome. Nucleic Acids Res. 32, D277-D280 (2004), <https://doi.org/10.1093/nar/gkh063>.
 * Kanehisa, M., Goto, S., Hattori, M., Aoki-Kinoshita, K.F., Itoh, M., Kawashima, S., Katayama, T., Araki, M., and Hirakawa, M.; From genomics to chemical genomics: new developments in KEGG. Nucleic Acids Res. 34, D354-357 (2006), <https://doi.org/10.1093/nar/gkj102>.
 * Kanehisa, M., Araki, M., Goto, S., Hattori, M., Hirakawa, M., Itoh, M., Katayama, T., Kawashima, S., Okuda, S., Tokimatsu, T., and Yamanishi, Y.; KEGG for linking genomes to life and the environment. Nucleic Acids Res. 36, D480-D484 (2008), <https://doi.org/10.1093/nar/gkm882>.
 * Kanehisa, M., Goto, S., Furumichi, M., Tanabe, M., and Hirakawa, M.; KEGG for representation and analysis of molecular networks involving diseases and drugs. Nucleic Acids Res. 38, D355-D360 (2010), <https://doi.org/10.1093/nar/gkp896>.
 * Kanehisa, M., Goto, S., Sato, Y., Furumichi, M., and Tanabe, M.; KEGG for integration and interpretation of large-scale molecular datasets. Nucleic Acids Res. 40, D109-D114 (2012), <https://doi.org/10.1093/nar/gkr988>.
 * Kanehisa, M., Goto, S., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; Data, information, knowledge and principle: back to metabolism in KEGG. Nucleic Acids Res. 42, D199â€“D205 (2014), <https://doi.org/10.1093/nar/gkt1076>.
 * Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; KEGG as a reference resource for gene and protein annotation. Nucleic Acids Res. 44, D457-D462 (2016), <https://doi.org/10.1093/nar/gkv1070>.
 * Kanehisa, Furumichi, M., Tanabe, M., Sato, Y., and Morishima, K.; KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res. 45, D353-D361 (2017), <https://doi.org/10.1093/nar/gkw1092>.

